src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -625,10 +679,10 @@ def save(self, new_comment=None, notify=True):
       if foundin_values:
         self._data['issueState']['foundInVersions'] = foundin_values
 
-      severity_text = _extract_label(self.labels, 'Security_Severity-')
-      logs.log('google_issue_tracker: severity_text: %s' % severity_text)
-      severity = _get_severity_from_crash_text(severity_text)
-      self._data['issueState']['severity'] = severity
+      severity = _get_severity_from_labels(self.labels) or _DEFAULT_SEVERITY
+      if severity is not None:
```

Comment:
severity will always be not None here.

Reply:
Oh right, missed this in the latest commit. Fixed.